src/prommis/solvent_extraction/tests/test_sx_model_dynamic.py

```diff
@@ -0,0 +1,262 @@
+        ) == pytest.approx(9.84e-08, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.aqueous[time_duration, 3].conc_mass_comp["La"]
+        ) == pytest.approx(4.77, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.aqueous[time_duration, 3].conc_mass_comp["Ce"]
+        ) == pytest.approx(0.055, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.aqueous[time_duration, 3].conc_mass_comp["Pr"]
+        ) == pytest.approx(0.00499, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.aqueous[time_duration, 3].conc_mass_comp["Nd"]
+        ) == pytest.approx(0.00016, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.aqueous[time_duration, 3].conc_mass_comp["Sm"]
+        ) == pytest.approx(5.707e-08, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.aqueous[time_duration, 3].conc_mass_comp["Gd"]
+        ) == pytest.approx(7.36e-05, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.aqueous[time_duration, 3].conc_mass_comp["Dy"]
+        ) == pytest.approx(1.584e-08, rel=1e-1)
+
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Al"]
+        ) == pytest.approx(4.04, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Ca"]
+        ) == pytest.approx(26.47, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Fe"]
+        ) == pytest.approx(0.783, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Sc"]
+        ) == pytest.approx(336.104, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Y"]
+        ) == pytest.approx(45.23, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["La"]
+        ) == pytest.approx(128.58, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Ce"]
+        ) == pytest.approx(315.71, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Pr"]
+        ) == pytest.approx(53.04, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Nd"]
+        ) == pytest.approx(144.92, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Sm"]
+        ) == pytest.approx(26.18, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Gd"]
+        ) == pytest.approx(12.315, rel=1e-1)
+        assert value(
+            m.fs.solex.mscontactor.organic[time_duration, 1].conc_mass_comp["Dy"]
+        ) == pytest.approx(7.17, rel=1e-1)
```

Comment:
GitHub appears to have lost my final comment: You should add a new test here to ensure there are no numerical warnings from the DiagnosticsToolbox (`assert_no_numerical_warnings()`).